dbclient/dbclient.py

```diff
@@ -14,19 +17,30 @@ def pprint_j(i):
     print(json.dumps(i, indent=4, sort_keys=True))
 
 
+def url_validation(url):
+    if '/?o=' in url:
+        # if the workspace_id exists, lets remove it from the URL
+        url = re.sub("\/\?o=.*", '', url)
```

Comment:
got it

Reply:
We need to remove `/` because the URL can only have one `/` in the API calls. 
We don't need to keep anything w/ `=o` since it's shared resources and our central services can identify workspaces by the token used, so it's not necessary. There was a few other cases where this helped and we should continue doing it.